test/errors/resolution/dotTypeOnType.2.good

```diff
@@ -0,0 +1,12 @@
+─── error in dotTypeOnType.chpl:4 [DotTypeOnType] ───
+  Can't apply '.type' to a type (int(64)).
```

Comment:
IMO we should quote this, and I don't have a problem with changing production output to match. Part of dyno's business is revamping error messages anyway.